packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -358,7 +358,8 @@ function Routes() {
         options={{
           searchStringToObject: parse,
           objectToSearchString: stringify,
-          updateType: 'replaceIn' // replace just a single parameter when updating query-state, leaving the rest as is
+          updateType: 'replaceIn', // replace just a single parameter when updating query-state, leaving the rest as is
```

Comment:
I think it's a good time to probably abstract out all these verbose options and their imports, and import this whole object from `useArbQueryParams` to keep only core business logic in `App.tsx`?

something like 
`import {queryParamOptions} from "useArbQueryParams"; and,
options={queryParamOptions}`

Reply:
We can also just export the entire provider which is already configured in the file.

```
export function ArbQueryParamsProvider({ children }: { children: React.ReactNode }) {
  return (
    <QueryParamProvider
      adapter={ReactRouter5Adapter}
      // and all the other props
    >
      {children}
    </QueryParamProvider>
  )
}
```